libretroshare/src/file_sharing/hash_cache.cc

```diff
@@ -117,8 +117,8 @@ void HashStorage::data_tick()
                 if(!mChanged)	// otherwise it might prevent from saving the hash cache
                 {
                     std::cerr << "Stopping hashing thread." << std::endl;
-                    shutdown();
-                    mRunning = false ;
+					askForStop();
```

Comment:
I don't agree - but this comes down to a matter of taste. Remove two tabs and it almost looks fine in the commit again 😸